
syntax = "proto3";

package helloworld;

service Greeter2 {
  rpc SayHello (HelloRequest) returns(HelloReply){}
}

message HelloRequest {
  string name = 1;
  repeated int32 tipe_data = 2;
  int32 size_all = 3;
  repeated int32 size_arr = 4;
  repeated int32 timeEpoch = 5;
  bytes datablob = 6;
}

message HelloReply {
  string name = 1;
  repeated int32 tipe_data = 2;
  int32 size_all = 3;
  repeated int32 size_arr = 4;
  repeated int32 timeEpoch = 5;
  bytes datablob = 6;
}

